drivers/volume/portworx/portworx.go

```diff
@@ -438,13 +440,29 @@ func (d *portworx) ValidateVolumeCleanup() error {
 	return d.schedOps.ValidateVolumeCleanup(d.nodeDriver)
 }
 
-func (d *portworx) StopDriver(n node.Node) error {
-	return d.nodeDriver.Systemctl(n, pxSystemdServiceName, node.SystemctlOpts{
-		Action: "stop",
-		ConnectionOpts: node.ConnectionOpts{
-			Timeout:         stopDriverTimeout,
+func (d *portworx) StopDriver(n node.Node, force bool) error {
+	var err error
+	if force {
+		pxCrashCmd := "sudo kill -9 px-storage"
+		_, err = d.nodeDriver.RunCommand(n, pxCrashCmd, node.ConnectionOpts{
+			Timeout:         crashDriverTimeout,
 			TimeBeforeRetry: defaultRetryInterval,
-		}})
+		})
+		if err != nil {
+			logrus.Warnf("failed to run cmd: %s. err: %v", pxCrashCmd, err)
```

Comment:
We should return if there is an error. Anyway the test will fail when we return the err at the end. It will save us the waiting time.